### ISSUE TITLE
Database login dropping

### ISSUE THREAD

**Issue by rob-baron** [CCI-MOC/xdmod-cntr#67]
try adjusting timeouts to keep the database login from dropping

**Comment by rob-baron** [comment]
Got this error message while running xdmod-ingestor:

        [error] Unable to connect to the database, please check the following: 
        - settings for 'database' in portal_settings.ini are correct.
        - the database identified by the 'database' section of portal_settings.ini is running, 
           accepting connections, and that the user specified has connection privileges.

which corresponds to this error message in '/var/log/xdmod/etl_overseer.log':

        'SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'mariadb' (110)'

**Comment by joachimweyl** [comment]
> Unable to connect to the database, please check the following:
This seems to be coming from [here](https://github.com/ubccr/xdmod/blob/ece9da28234f24021fc6e68b63399ee67cee348e/bin/acl-config#L249).

**Comment by pjd-nu** [comment]
It's pretty hard to get ETIMEDOUT (error 110) from Linux TCP. It looks like it typically happens when SYN packets are getting firewalled, with a typical timeout of 120s. (possibly less if the kernel knows the round-trip time to the destination is low)

In particular, I was totally wrong about the potential for ETIMEDOUT if server threads are all busy. I ran a quick test, and the client will happily connect and even write data to the socket. 

A quick grep through the linux kernel and GlibC source doesn't show many cases where ETIMEDOUT is returned. But... most of the cases in glibc are pthreads-related (e.g. wait on condvar with timeout) so it might in fact be coming from an application-level timeout.

**Comment by joachimweyl** [comment]
@rob-baron is this resolved by the same manifest label resolution as https://github.com/OCP-on-NERC/operations/issues/2

**Comment by rob-baron** [comment]
@joachimweyl - yes this is resolved as @larsks decided to create a separate issue is a more central place than this issue as it was created when the problem was first encountered.  I am currently working on testing this change as this project has 2 services that need to be uniquely connected to 2 different containers.
